.sonarcloud.properties

```
@@ -0,0 +1,15 @@
+# Path to sources
+#sonar.sources=.
+#sonar.exclusions=
+#sonar.inclusions=
+
+# Path to tests
+sonar.tests=app/tests/**, app/cypress/**
```

Could it be that globbing isn't supported here?
```suggestion
sonar.tests=app/tests,app/cypress
```

---

...are these valid root-relative paths? I don't see an `app` folder. Maybe this should have simply been
```suggestion
sonar.tests=tests,cypress
```

---

I can't find documentation on this (I have links but Sonar deleted their docs?) but I'm recalling something about the `sonar.tests` parameter only working to identify the top level directory of tests and needing to use includes/excludes for globbing if needed